Ago-Dic-2020/aguirre-juarez-david/practica-1/matrix.py

```diff
@@ -0,0 +1,35 @@
+class Matrix:
+	__matrix = []
```

Comment:
No es necesario declarar la matriz acá. Esto debido a que queremos que sea variable de instancia y no de clase: https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3